components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -2049,33 +2058,17 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
     public async getTeam(ctx: TraceContext, teamId: string): Promise<Team> {
         traceAPIParams(ctx, { teamId });
 
-        if (!uuidValidate(teamId)) {
-            throw new ResponseError(ErrorCodes.BAD_REQUEST, "organization ID must be a valid UUID");
-        }
-
         this.checkAndBlockUser("getTeam");
 
-        const team = await this.guardTeamOperation(teamId, "get");
-        if (!team) {
-            throw new ResponseError(ErrorCodes.NOT_FOUND, `Team ${teamId} does not exist`);
-        }
```

Comment:
The `guardTeamOperation` would've already thrown an error, so it wouldn't actually ever reach the `if` after